components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -111,7 +113,7 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 	}, nil
 }
 
-func (b *ProvisionEndpoint) validateAndExtract(details domain.ProvisionDetails) (internal.ERSContext, internal.ProvisioningParametersDTO, error) {
+func (b *ProvisionEndpoint) validateAndExtract(details domain.ProvisionDetails, logger logrus.FieldLogger) (internal.ERSContext, internal.ProvisioningParametersDTO, error) {
```

Comment:
can we validate here the `kymaVersion` input when it exist? 
right now when we pass a wrong version e.g `master-87c578b` instead of `master-87c578b4`, KEB proceeds with processing and fails on Initialization step without any details:
```
127.0.0.1 - - [30/Mar/2020:08:41:43 +0000] "PUT /v2/service_instances/78810f43-fb38-4db6-845d-bff85b27daaa HTTP/1.1" 202 53
time="2020-03-30T08:41:43Z" level=info msg="Start process operation steps" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15
time="2020-03-30T08:41:43Z" level=info msg="Start step" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=info msg="instance not exist, initialize runtime input request" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=info msg="setting up input builder to work with master-87c578b Kyma version" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=info msg="create input creator for \"ca6e5357-707f-4565-bbbd-b3ab732597c6\" plan ID" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=error msg="cannot create input creator for plan ca6e5357-707f-4565-bbbd-b3ab732597c6" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=error msg="Process operation failed: cannot create provisioning input creator" instanceID=78810f43-fb38-4db6-845d-bff85b27daaa operation=40879dc3-233b-46f0-a99c-9a558de8ec15 step=Initialization
time="2020-03-30T08:41:43Z" level=error msg="Error from process: cannot create provisioning input creator"
time="2020-03
```

Reply:
The log is added, info about bad hash will be there. After discussion, the validation will not be added.